plugin/hosts/metrics.go

```diff
@@ -0,0 +1,32 @@
+package hosts
+
+import (
+	"github.com/coredns/coredns/plugin"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	// DeprecatedhostsEntries is the combined number of entries in hosts and Corefile.
+	// non-histogram and non-summary metrics should not have "_count" suffix
+	DeprecatedhostsEntries = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: plugin.Namespace,
+		Subsystem: "hosts",
+		Name:      "entries_count",
+		Help:      "[]The combined number of entries in hosts and Corefile.",
+	}, []string{})
+	// HostsEntries is the combined number of entries in hosts and Corefile.
+	HostsEntries = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: plugin.Namespace,
+		Subsystem: "hosts",
+		Name:      "entries_size",
```

Comment:
This should just be "entries", because this is the "unit". "size", just like "count", is superfluous.

```suggestion
		Name:      "entries",
```